TicketSwap/solar
# Improve Select onKeyDown events

# Description

A native `<select>` element has the ability to take in input from the user using a keyboard, for example for a11y reasons. The Select component was not able to handle this, only escape / space and key down / key up events. This PR adds support for search inputs in a select dropdown. The query you type in as a user resets after 1 seconds.

I noticed the `<Toast />` component wasn't in a category (must have missed it the other PR) so I also added it to the right category.

## How to test

- Checkout this branch
- `$ yarn storybook`
- Go to `<Select />` component
- See if you can search on the countries in the list by typing in a few letters of the country you're looking for
